src/components/CreateSecret/Form/UniversalFields.js

```diff
@@ -73,13 +74,24 @@ const UniversalFields = props => {
         })}
         disabled={loading}
       />
-      <FormGroup
-        legendText={intl.formatMessage({
-          id: 'dashboard.universalFields.secretType',
-          defaultMessage: 'Type'
-        })}
-        disabled={loading}
-      >
+      <FormGroup disabled={loading}>
+        <Tooltip
```

Comment:
You can set the tooltip as the legendText. Having the legend is important for accessibility so that a screenreader can announce the correct group regardless of tab direction, i.e. if a user tabs backwards from the username input to the selected radio button it will announce the group + selection.

```js
      <FormGroup
        disabled={loading}
        legendText={
          <Tooltip
            direction="right"
            tabIndex={0}
            ...
```

Reply:
Thanks, pushed